vmdb/app/models/vmdb_database_connection.rb

```diff
@@ -1,15 +1,27 @@
-class VmdbDatabaseConnection < ActsAsArModel
-  set_columns_hash(
-    :address           => :string,
-    :application       => :string,
-    :blocked_by        => :integer,
-    :command           => :string,
-    :spid              => :integer,
-    :task_state        => :string,
-    :wait_resource     => :string,
-    :wait_time         => :integer,
-    :vmdb_database_id  => :integer
-  )
+class VmdbDatabaseConnection < ActiveRecord::Base
+  class << self
+    attr_accessor :aar_columns
+  end
+  self.aar_columns = []
+
+  self.table_name = 'pg_stat_activity'
+  self.primary_key = nil
+
+  has_many :pg_locks, :primary_key => 'pid', :foreign_key => 'pid'
```

Comment:
Oh but then you'll have to rename this too...hmmm... I'm so conflicted :stuck_out_tongue: